dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -7,6 +7,7 @@
 from __future__ import print_function
 from __future__ import division
 
+import pandas
```

Comment:
The convention throughout the library, and somewhat standard, is to `import pandas as pd` and use `pd` whenever invoking a pandas function.

Reply:
Also, you're not invoking any pandas function here, is this import even necessary?